### PR TITLE
Issue #122: Add default values for connector variables 

### PR DESCRIPTION
**Pull request by CherfaElyes**
* Added connector variables default values on `LinuxService`, `LinuxProcess`, `WindowsService` & `WindowsProcess`.
* Tested these connectors on MetricsHub Agent.

# Testing on WindowsProcess

Issue #122 Community Connectors

Tested all the modified connectors as well, everything worked perfectly!

### CODE REVIEW

**Review by NassimBtk — COMMENTED** [review]
I am fine with that, probably the descriptions needs to be reviewed as they will go to the official documentation.

**Comment by bertysentry** [comment]
Now this need to be displayed in the Connector Directory!

**Comment by NassimBtk** [comment]
Yes @bertysentry 😉currently @CherfaElyes is working on that.

File: src/main/connector/system/LinuxProcess/LinuxProcess.yaml [hunk 1]
```diff
@@ -18,6 +18,16 @@ connector:
       expectedResult: /bin/ps
       errorMessage: Not a valid Linux host.
     tags: [ system, linux ]
+  variables:
+    matchName:
+      description: Regular expression pattern to match process names to monitor.
```
Comment (iguitton) [inline src/main/connector/system/LinuxProcess/LinuxProcess.yaml:23]: description: Regular expression pattern to match process names for monitoring.

**Review by iguitton — COMMENTED** [review]
@CherfaElyes , @NassimBtk : I made a suggestion for improvement.

**Comment by CherfaElyes** [comment]
My pleasure @iguitton

**Review by NassimBtk — APPROVED** [review]

**Pull request merged**